package.json

```diff
@@ -11,6 +12,7 @@
   },
   "private": true,
   "scripts": {
+    "build-js-api": "esbuild --bundle src/bun.js/src/*.exports.js --outdir=src/bun.js/dist/ --format=esm --platform=node",
```

Comment:
Should I add a watch version: `build-js-api:dev`?

Is `build-js-api` too wordy?

Reply:
Maybe just `build:js`?